src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenReadonlyStructTests.cs

```diff
@@ -2065,6 +2065,8 @@ .maxstack  2
         public void ReadOnlyMethod_CallReadOnlyMethodOnField()
         {
             var csharp = @"
+#pragma warning disable CS0649
```

Comment:
why?

Reply:
Flow analysis used to see `s1.M1()` as a write to `s1`, and now that it doesn't it warns about the unused field `s1`.

Reply:
i'd prefer to have the test just list that diagnostic FWIW.